Source/WebCore/Modules/WebGPU/GPUCommandEncoder.cpp

```diff
@@ -134,7 +134,14 @@ static PAL::WebGPU::CommandBufferDescriptor convertToBacking(const std::optional
 
 Ref<GPUCommandBuffer> GPUCommandEncoder::finish(const std::optional<GPUCommandBufferDescriptor>& commandBufferDescriptor)
 {
-    return GPUCommandBuffer::create(m_backing->finish(convertToBacking(commandBufferDescriptor)));
+    auto result = GPUCommandBuffer::create(m_backing->finish(convertToBacking(commandBufferDescriptor)));
+    m_backing->destroy();
```

Comment:
Is this right? See my other comment about how the browser objects need to still exist for validation, state tracking, etc.